### DISCUSSION TITLE
NAPS2 8.0 Beta Feedback

### DISCUSSION THREAD

**Discussion by cyanfish** [cyanfish/naps2#467] [Announcements]
NAPS2 8.0 (beta) has been released! There are significant under-the-hood changes (especially on Windows) so let me know if you experience issues after upgrading.

Some highlights:
- New sidebar UI
- High-dpi support
- Dark mode support on Windows
- Windows 7 is no longer supported

[Downloads](https://www.naps2.com/download)
[Changelog](https://www.naps2.com/changelog)

Known issues:
- ~~First-time startup is a bit slow~~ (fixed in beta 2)

**Comment by krystofkrticka** [comment]
New sidebar can be confusing when you set Use native UI in your profile. But it still displays settings which you can predefine.

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Comment by YanJi314** [comment]
Thanks for the update and the new sidebar UI is wonderful.

However, when I use WIA as the driver, nothing happens after the progress bar ended. The exact scanner model is `Epson Perfection V370`. (Already tried turning off `Exclude blank pages` but the problem persisted.)

![image](https://github.com/user-attachments/assets/e541f44d-bc7d-4758-a09f-bc15d4135c4d)

Expected behavior: the scanned images should appear in the window.

I found those messages in `AppData\Roaming\NAPS2\errorlog.txt` but no extra information is provided
```
2024-10-07 10:26:59.4930 10376 Ignoring empty stream from WIA 
2024-10-07 10:27:27.1689 10376 Ignoring empty stream from WIA 
2024-10-07 10:27:56.8103 10376 Ignoring empty stream from WIA 
2024-10-07 10:28:51.5764 10376 Ignoring empty stream from WIA 
2024-10-07 10:29:19.8437 10376 Ignoring empty stream from WIA 
```


The same configuration can work in version 7.5.2.0
![image](https://github.com/user-attachments/assets/c86af365-6d41-4737-8df5-2dcc40e387b7)

OS: Windows 11 Pro Workstation 23H2

**Reply by camheelio** [reply]
The same for me with Canon CanoScan LiDE 400 and WIA Driver. But only when "use predefined settings" is selected. With "Use native UI" the page is added to the UI.

Windows 11 24H2

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Comment by Tarek-Hasan** [comment]
So, now someone can add a profile from three places in the main window! I don't mind "Profiles" button. Also like the new profile sidebar, which gives a modern look to the app. But in the "Scan" button's drop-down menu, there's a "New Profile" option. It doesn't add any value other than visual clutter. You should remove this repetitive option from the "Scan" menu.

**Comment by folknor** [comment]
Hi!

Lower-right "notifications" are never cleared in GTK. Also the "Running OCR..." box never goes away even after 10 minutes, but when I save the PDF everything is OCR'd.

EDIT: Also the X button to close notification doesn't work.

![image](https://github.com/user-attachments/assets/0407bf25-ccf8-44d7-bb74-ebbd409b2294)

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Comment by mglsj** [comment]
Hi, drag and dropping pdf files to naps2 window isn't seem to be working on Windows 11 (exe installation).

**Comment by coderuk66** [comment]
After cropping an image the view is updated and the image looks like its cropped incorrectly. its not showing the full image. when the image is saved to a file it looks as I cropped it (thats the good bit).
I like the side bar view. It can speed things up when checking the profile settings.

**Comment by TopstarBluecoat** [comment]
On macOS 14.7: After the scanning process has finished, no preview of the scanned document is shown in the NAPS application window. NAPS 7.5.1 was the last version when this worked. I noticed the problem in 7.5.2 and now in 8.0 beta. So I went back to 7.5.1

**Reply by cyanfish** [reply]
Can you follow [these instructions](https://www.naps2.com/doc/troubleshooting#debug-log) to collect debug logs with 7.5.2?

**Reply by TopstarBluecoat** [reply]
I did a scan with 7.5.2 while debug mode was on. Here's the result:

2024-10-10 18:32:16.2358 57117 Scanning with ScanDevice { Driver = Apple, ID = DF3589C1-DF35-89C1-DF35-89C1DF3589C1, Name = Samsung SCX-3400 Series, IconUri =  } 
2024-10-10 18:32:16.3645 57117 Scan source: Feeder; bit depth: Color; dpi: 200; page size: 210x297 mm 
2024-10-10 18:32:16.4850 57118 ICC: Opening session 
2024-10-10 18:32:18.7527 57118 DidOpenSession (null) 
2024-10-10 18:32:18.7569 57118 ICC: Waiting for ready 
2024-10-10 18:32:18.9124 57118 DidBecomeReady 
2024-10-10 18:32:18.9135 57118 ICC: Selecting unit 
2024-10-10 18:32:18.9333 57118 DidSelectFunctionalUnit ICScannerFunctionalUnitFlatbed (null) 
2024-10-10 18:32:18.9352 57118 ICC: Setting scan parameters 
2024-10-10 18:32:18.9419 57118 ICC: Requesting scan 
2024-10-10 18:32:31.6724 57118 DidScanToBandData buffer complete 
2024-10-10 18:32:31.6777 57118 Image data: width 1654, height 2338, type Rgb, comp 3, bits/comp 8, bits/pixel 24, bytes/row 4962, data len 11601156 
2024-10-10 18:32:31.6778 57118 Flushing image with color sync profile /System/Library/Frameworks/ICADevices.framework/Versions/A/Resources/Scanner RGB Profile.icc 
2024-10-10 18:32:31.6974 57118 Setting resolution to 200 
2024-10-10 18:32:31.9225 57118 DidCompleteScan (null) 
2024-10-10 18:32:31.9238 57118 ICC: Waiting for scan results 
2024-10-10 18:32:31.9239 57118 ICC: Closing session 
2024-10-10 18:32:31.9239 57118 DidCloseSession (null) 
2024-10-10 18:32:31.9248 57118 ICC: Scan success

**Reply by TopstarBluecoat** [reply]
It looks like the problem has something to do with the profiles. If I use a profile for greyscale or black/white then the preview of the document is shown in NAPS. But when I use a color profile, it does not work.

**Reply by stevemagruder** [reply]
This happens for me in Windows 11 24H2 for 8.0b1, although it works fine in 7.5.2.

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Reply by stevemagruder** [reply]
Fixed for me. Thanks!

**Comment by funkerwolf** [comment]
Just a great update, long time ago should have made such an interface.  All that's left is to build a UWP version and update the design for Windows 11, it will be just awesome. But still there is a request, can you work on compatibility with Kyocera MFP? Very lacking in their support of TWAIN implementation, have to.... to use winscan2pdf instead of Naps2 to work with them. An example of such a MFP model FS-1025, very common in view of its cheapness and reliability, drivers (https://www.kyoceradocumentsolutions.com/asia/download/software/FS-1020_1025_1120_1125MFP_v35.zip) for it there are providing scanning over the network, just not all of them can not find. 
And when will the ability to run as a service be implemented, so as not to keep the naps2 window running itself?

**Comment by coderuk66** [comment]
Some suggestions.

- In the sidebar profile view can we have the keyboard shortcut key (F function) shown? perhaps under the profile name.

- Is there a keyboard shortcut key for "Scan"? to scan using the profile selected in the sidebar. Prefer to just tap a key sometimes than reach for my mouse.

quirk/bug: After scanning the screen briefly switches to another window's app then back to show naps2.

**Comment by funkerwolf** [comment]
In scan mode via WIA on HP MFP M125 the result is not output, via twain is fine. The same on USB and Network connection. Checked on other HP models, it looks like scanning through WIA is broken. Rolled back to 7.5.2 via GPO with the 8th one removed. Everything is normal now.
[debuglog.txt](https://github.com/user-attachments/files/17351902/debuglog.txt)
[errorlog.txt](https://github.com/user-attachments/files/17351904/errorlog.txt)

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Comment by coderuk66** [comment]
Oddly after I removed another program I needed to reinstall, NAPS2 now starts so much quicker. its x4 faster startup (like it should be). see if stays that way. I'm using the portable installation (naps2-8.0b1-win-x64).
maybe installing another app fixed issues with the registry that were caused by naps? hence why its now starting lightning fast. can other users try this out see if it improves start up speed? just install an app or uninstall an app then reinstall the app. backup your app's settings / data before doing so of course.

**Reply by cyanfish** [reply]
The slow startup is related to Windows Defender scanning. In my testing it only happens immediately after installing NAPS2 or rebooting, subsequent starts should be fast.

I'm hoping that as the NAPS2 code signing certificate generates more reputation this might fix itself at some point. Or if you're so inclined you can add the NAPS2 folder to the Defender exclusion list.

**Reply by coderuk66** [reply]
makes sense. I only start up naps once and leave it open so a slow startup isn't much of an issue. I've gone back to 7.5.2 for now until the next beta release. I'm using Avira Free Security as my AV software. Was on the beta 1 for a couple of weeks just thought it odd when i saw it start up so fast as its never been that fast before.

**Reply by cyanfish** [reply]
Slow startup should be fixed in [beta 2](https://www.naps2.com/download), though at the cost of a larger installer which I might try and fix down the road.

**Comment by P12P** [comment]
$ naps2
Unhandled exception. System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> Autofac.Core.DependencyResolutionException: An exception was thrown while activating NAPS2.EtoForms.Ui.GtkDesktopForm.
 ---> Autofac.Core.DependencyResolutionException: An exception was thrown while invoking the constructor 'Void .ctor(NAPS2.Config.Naps2Config, NAPS2.EtoForms.Desktop.DesktopKeyboardShortcuts, NAPS2.EtoForms.Notifications.NotificationManager, NAPS2.Util.CultureHelper, NAPS2.EtoForms.ColorScheme, NAPS2.Config.IProfileManager, NAPS2.Images.UiImageList, NAPS2.Images.ThumbnailController, NAPS2.Images.UiThumbnailProvider, NAPS2.EtoForms.Desktop.DesktopController, NAPS2.EtoForms.Desktop.IDesktopScanController, NAPS2.EtoForms.Desktop.ImageListActions, NAPS2.EtoForms.Widgets.ImageListViewBehavior, NAPS2.EtoForms.Desktop.DesktopFormProvider, NAPS2.EtoForms.Desktop.IDesktopSubFormController, System.Lazy`1[NAPS2.EtoForms.Ui.DesktopCommands], NAPS2.EtoForms.IDarkModeProvider, NAPS2.EtoForms.Desktop.Sidebar, NAPS2.EtoForms.IIconProvider)' on type 'GtkDesktopForm'.
 ---> GLib.GException: Нераспознанный формат файла изображения
   at Gdk.PixbufLoader.Write(Byte[] buf, UInt64 count)
   at Gdk.PixbufLoader.Write(Byte[] bytes, UInt32 count)
   at Gdk.PixbufLoader.LoadFromStream(Stream input)
   at Gdk.PixbufLoader.InitFromStream(Stream stream)
   at Gdk.PixbufLoader..ctor(Stream stream)
   at Gdk.Pixbuf..ctor(Stream stream)
   at Eto.GtkSharp.Drawing.IconHandler.Create(Stream stream)
   at Eto.Drawing.Icon..ctor(Stream stream)
   at NAPS2.EtoForms.EtoExtensions.ToEtoIcon(Byte[] bytes)
   at NAPS2.EtoForms.Ui.DesktopForm..ctor(Naps2Config config, DesktopKeyboardShortcuts keyboardShortcuts, NotificationManager notificationManager, CultureHelper cultureHelper, ColorScheme colorScheme, IProfileManager profileManager, UiImageList imageList, ThumbnailController thumbnailController, UiThumbnailProvider thumbnailProvider, DesktopController desktopController, IDesktopScanController desktopScanController, ImageListActions imageListActions, ImageListViewBehavior imageListViewBehavior, DesktopFormProvider desktopFormProvider, IDesktopSubFormController desktopSubFormController, Lazy`1 commands, Sidebar sidebar, IIconProvider iconProvider)
   at NAPS2.EtoForms.Ui.GtkDesktopForm..ctor(Naps2Config config, DesktopKeyboardShortcuts keyboardShortcuts, NotificationManager notificationManager, CultureHelper cultureHelper, ColorScheme colorScheme, IProfileManager profileManager, UiImageList imageList, ThumbnailController thumbnailController, UiThumbnailProvider thumbnailProvider, DesktopController desktopController, IDesktopScanController desktopScanController, ImageListActions imageListActions, ImageListViewBehavior imageListViewBehavior, DesktopFormProvider desktopFormProvider, IDesktopSubFormController desktopSubFormController, Lazy`1 commands, IDarkModeProvider darkModeProvider, Sidebar sidebar, IIconProvider iconProvider)
   at lambda_method2(Closure, Object[])
   at Autofac.Core.Activators.Reflection.BoundConstructor.Instantiate()
   --- End of inner exception stack trace ---
   at Autofac.Core.Activators.Reflection.BoundConstructor.Instantiate()
   at Autofac.Core.Activators.Reflection.ReflectionActivator.<>c__DisplayClass14_0.<UseSingleConstructorActivation>b__0(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Middleware.DelegateMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Resolving.Middleware.DisposalTrackingMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Resolving.Middleware.ActivatorErrorHandlingMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   --- End of inner exception stack trace ---
   at Autofac.Core.Resolving.Middleware.ActivatorErrorHandlingMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Pipeline.ResolvePipeline.Invoke(ResolveRequestContext context)
   at Autofac.Core.Resolving.Middleware.RegistrationPipelineInvokeMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Resolving.Middleware.SharingMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Resolving.Middleware.CircularDependencyDetectorMiddleware.Execute(ResolveRequestContext context, Action`1 next)
   at Autofac.Core.Resolving.Pipeline.ResolvePipelineBuilder.<>c__DisplayClass14_0.<BuildPipeline>b__1(ResolveRequestContext context)
   at Autofac.Core.Pipeline.ResolvePipeline.Invoke(ResolveRequestContext context)
   at Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, ResolveRequest& request)
   at Autofac.Core.Resolving.ResolveOperation.ExecuteOperation(ResolveRequest& request)
   at Autofac.Core.Resolving.ResolveOperation.Execute(ResolveRequest& request)
   at Autofac.Core.Lifetime.LifetimeScope.ResolveComponent(ResolveRequest& request)
   at Autofac.Core.Lifetime.LifetimeScope.Autofac.IComponentContext.ResolveComponent(ResolveRequest& request)
   at Autofac.ResolutionExtensions.TryResolveService(IComponentContext context, Service service, IEnumerable`1 parameters, Object& instance)
   at Autofac.ResolutionExtensions.ResolveService(IComponentContext context, Service service, IEnumerable`1 parameters)
   at Autofac.ResolutionExtensions.Resolve(IComponentContext context, Type serviceType, IEnumerable`1 parameters)
   at Autofac.ResolutionExtensions.Resolve[TService](IComponentContext context, IEnumerable`1 parameters)
   at Autofac.ResolutionExtensions.Resolve[TService](IComponentContext context)
   at NAPS2.AutofacFormFactory.Create[T]()
   at NAPS2.EntryPoints.GuiEntryPoint.Run(String[] args, Module imageModule, Module guiModule)
   at NAPS2.EntryPoints.GtkEntryPoint.Run(String[] args)
   at System.RuntimeMethodHandle.InvokeMethod(Object, Void**, Signature, Boolean)
   at System.Reflection.MethodBaseInvoker.InterpretedInvoke_Method(Object obj, IntPtr* args)
   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object, Span`1, BindingFlags)
   --- End of inner exception stack trace ---
   at System.Reflection.MethodBaseInvoker.InvokeDirectByRefWithFewArgs(Object, Span`1, BindingFlags)
   at System.Reflection.MethodBaseInvoker.InvokeWithOneArg(Object, BindingFlags, Binder, Object[], CultureInfo)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object, BindingFlags, Binder, Object[], CultureInfo)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at NAPS2.Program.Main(String[] args)
Аварийный останов (образ памяти сброшен на диск)
fedora41 x64

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Comment by regs01** [comment]
Cannot receive multuple pages from ScanJet N6010. Scanner does scan all pages in tray, but NAPS staying at scanning page 1 and in the end says no pages in tray. And no scanned pages appears.

Debug log only shows this
```
2024-11-13 14:11:01.6890 10416 Scanning with ScanDevice { Driver = Wia, ID = {6BDD1FC6-810F-11D0-BEC7-08002BE2092F}\0005, Name = HP Scanjet N6010, IconUri =  } 
2024-11-13 14:11:01.6890 10416 Scan source: Feeder; bit depth: Color; dpi: 200; page size: 210x297 mm 
2024-11-13 14:11:19.0037 10416 Ignoring empty stream from WIA 
2024-11-13 14:11:36.0812 10416 Ignoring empty stream from WIA 
2024-11-13 14:11:36.1197 10416 Ignoring empty stream from WIA 
```

It goes ok, if checking Use own interface. But that's an extra step.

**Reply by cyanfish** [reply]
This should be fixed in [beta 2](https://www.naps2.com/download).

**Comment by Basz001** [comment]
Hi,

(Version 8. 1 beta, portable.)  Periodically I get a message (Twain) that there is no connection to the scanner (Brother DS 940DW) that I should check the cable, etc.
This is followed by a message that Nasp is not responding and then crashes.
Tried everything (settings) however no result. Problem remains.
I  reinstalled version 7.5. 2 again (portable) ... No problems... :-) 
In the discussion I read that someone else also has a problem with the 'connection'.
I have Windows 11 24H2.

Have a great time!

**Reply by coderuk66** [reply]
stick with 7.5.2 and wait till next beta release

**Reply by Basz001** [reply]
Thank you for your response! 
I wish you success and want to thank you for all the great work you do! 
Happy days!

**Reply by cyanfish** [reply]
Can you try again with [beta 2](https://www.naps2.com/download)?

**Reply by Basz001** [reply]
Hi,
Beta2:

It is not going smoothly at all yet. Some error messages.
I have the strong impression that the problems are rather related to Windows 11 _24H2_ ?!!
In version 23 the order was connect scanner, start Naps and scan. No problems at all. 
Now I hear a beep and see "open Devices and Printers" with some dots below (...) 
I have to press the on/off button of the scanner to activate the scanner. 
Now in place of the dots appears : eject DS-940DW. (OK :)
When creating a first twain profile I immediately got an error message. (see image) Creating a WIA profile is better, scanning too.
Once scanning gets going after some error messages (Twain error: Nods), removing, switching my scanner off and on again, Nasp (beta) and 7.5.3 work fine.
I checked the scanner properties: this device is working **correctly**, status device ready \\.\usbscan0.
Reinstalled the driver. 
Again, in Windows 23H I didn't have these problems. 
Maybe Naps can do some sort of check to see if the scanner is ON ??? 
Sometimes I would get a message that the scanner was busy, but it wasn't scanning at all. 
I could close Naps without a crash.
In short: it's a bit of _trial and error_ for me now.
Once scanning gets going, it's fine .....
Good luck with everything!

Bas



![Schermafbeelding 2025-01-02 150801](https://github.com/user-attachments/assets/02d52a07-216f-4724-a646-548dc639d863)

**Comment by coderuk66** [comment]
thanks for the update.

**Comment by funkerwolf** [comment]
Too bad it's too late to get a fix, our organization decided to fold. But it was possible to make it more convenient for all our colleagues earlier.

**Comment by camheelio** [comment]
Beta 2 is working with Canon CanoScan Lide 400 WIA on Windows 11 24H2

**Comment by akuropka** [comment]
Using a Kyocera Ecosys M5521cdw with TWAIN.
Despite selecting A4 the page is always cut to s small strip.
Does not happen with v7.
Please see results below:
v8
![0005](https://github.com/user-attachments/assets/74187101-c04e-4cd1-95f6-791af104cd96)
v7
![0006](https://github.com/user-attachments/assets/5cf3c3b6-d90c-473a-a9f2-076bf90fe9be)

**Reply by cyanfish** [reply]
In your profile settings, under Advanced, Twain Implementation, try selecting "Native Transfer".

**Reply by akuropka** [reply]
Switched from x64 to Native Transfer but the result is the same.
![20250217_172830_NAPS2](https://github.com/user-attachments/assets/8501db79-3e79-4f13-9fd3-3cfe29644011)

**Reply by Tarek-Hasan** [reply]
Hi,
I'm just curious, are you using latest Twain drivers v3.2? Also, are you on win10 or win11?

**Reply by akuropka** [reply]
Windows 11

[Module]
Brand=Kyocera
Year=2019
Version=3.0.1421

It works with the stable branch...

**Reply by Tarek-Hasan** [reply]
If it's possible, please update to the latest driver package for your scanner and try again.

**Reply by akuropka** [reply]
Oh yes, I can do that, this would be 3.2.0512. However, this ends up in an endless number of one scanned immage... 
(and yet stable branch has no issue with this driver)

https://github.com/user-attachments/assets/4d038d91-5ecc-479d-b82a-ff942780b148

**Reply by Tarek-Hasan** [reply]
Sorry to bother you. Is this problem persists on both x64 and native twain mode with new driver?

**Reply by akuropka** [reply]
I'm happy if I can assist.
It doesn't matter which mode I select, the result is always the same.
I also tried deleting and setting up the profile again with no effect either.

**Reply by Tarek-Hasan** [reply]
Hi, did you try the latest version? v8.1.3 supposed to fix the issue.

**Reply by akuropka** [reply]
Yes, 8.1.3 fixed it.
https://github.com/cyanfish/naps2/issues/577

**Comment by coderuk66** [comment]
appreciate the update. 8.0b3

**Comment by akuropka** [comment]
https://github.com/cyanfish/naps2/discussions/467#discussioncomment-11750165
Fyi, this issue persists with b3.

**Comment by twwn** [comment]
(I'm using Linux/gtk3.)

1. **"Save PDF" & "Save Images" actual save dialogs:** Both ignore the chosen Default File Path. Instead, the dialog opens with the system's root folder selected and the placeholders not evaluated for the suggested filename. On actually saving, the placeholders are nonetheless properly evaluated; it would be more natural to show this evaluated name already in the dialog. Example: Setting `/tmp/pdf/test-$(mm)_$(ss).pdf` as DFP has the dialog effectively present saving `/test-$(mm)_$(ss).pdf` instead: If I then manually write `/tmp/test-$(mm)_$(ss).pdf` the saved file will be eg. `/tmp/test-38_01.pdf`.
2. **Profiles dialog:** New setups would be easier if there was a *"copy profile"* option.
3. ***"Single page file"* option:** It's is not self-explanatory resp. leaves one confused about its meaning.
4. ***"Skip save prompt":*** Could likewise use a word or two about when this suppression is triggered.
5. ***Email PDF*'s *"PDF Setting"*:** Rather than reuse *"Save PDF"*'s settings, it might be more useful to offer a separate set (to allow setting encryption, later perhaps lower quality, etc.). The aforementioned two settings and *Default File Path* don't make sense there anyway and so are confusing.
6. **PDF compatibility:** Would it be possible to add the newer A-4, or a *"newest"* setting to auto-apply it with future updates?
7. ***Save Images*'s *Default File Path*:** There is no `$(filetype)` placeholder. Likewise, in the actual save dialog, there's no auto-append of the extension when a format is chosen there.
8. ***Save Images*:** Since there's no separate option to pre-select its filetype, it might be worth to add a note that manually appending the relevant extension to that Default File Path is the supposed way to do this.
9. **New Outlook app:** Seems like there's no integration  yet?
10. **Image file formats:** Would be neat if WEBP & AVIF were added.
